charts/agent/templates/_helpers.tpl

```diff
@@ -318,3 +318,31 @@ Use global sysdig tags for agent
         {{- end -}}
     {{- end -}}
 {{- end -}}
+
+{{/*
+Determine Sysdig Secure features that need to be enabled/disabled
+*/}}
+{{- define "agent.secureFeatures" }}
+    {{- $secureBlockConfig := dict "security" (dict
+        "enabled" .Values.secure.enabled
```

Comment:
May want to explain why secure.enabled = true and false are not mirror images.
For secure.enabled, only security.enabled is set to true (along with the k8s_audit_server_enabled set to whatever the auditLog.enabled is)
For secure.enabled=false, all known secure features are disabled.

The explanation is that for secure.enabled=true we depend on the backend to enable the appropriate secure features for the customer. And since we don't know a priori what secure features may be enabled when secure.enabled=true we disable them all when secure.enabled=false

Reply:
Done!

Reply:
Thanks